### ISSUE TITLE
support required for swift 4.2

### ISSUE THREAD

**Issue by hr147** [devxoul/Toaster#139]
Please fix library for swift 4.2

**Comment by ZaidSA** [comment]
@devxoul Any updates on this?

**Comment by Coeur** [comment]
@devxoul what are your plans for Toaster?

**Comment by anonym24** [comment]
@hujiaju doesn't work
![image](https://user-images.githubusercontent.com/8851301/55115804-110cc980-50ee-11e9-9212-1ac56b0f5ab7.png)

**Comment by Coeur** [comment]
@anonym24 support for Swift 4.2 (and Swift 5.0) is available on the master branch, so you can simply do:
```ruby
pod 'Toaster', :git => 'https://github.com/devxoul/Toaster', :branch => 'master'
```

**Comment by anonym24** [comment]
@Coeur well there is still issue about Objective-C

**Comment by Coeur** [comment]
@anonym24 then it's not the right issue: you may want to create a separate issue, showing your Podfile, Podfile.lock and eventually a sample project demonstrating the problem. Also make sure you have [`use_frameworks!`](https://guides.cocoapods.org/syntax/podfile.html#use_frameworks_bang) or explain why you don't.

**Comment by devxoul** [comment]
Toaster doesn't support Objective-C since 2.0.0. (#89) You may create a bridge class to support Objective-C in your project.

```swift
import Toaster

 @objc class MyToast: NSObject {
  private let toast: Toast

   @objc init(text: String, delay: TimeInterval, duration: TimeInterval) {
    self.toast = Toast(text: text, delay: delay, duration: duration)
  }

   @objc func show() {
    self.toast.show()
  }
}
```

```objc
#import "MyApp-Swift.h"

[[[MyToast alloc] initWithText:@"Hello, world!" delay:0 duration:2.5] show];
```

**Comment by devxoul** [comment]
We can continue this thread on the duplicate: #143

**Comment by Coeur** [comment]
Version 2.2.0 was released and is compatible Swift 5.0 and Swift 4.2.
